include/gcodeExport.h

```diff
@@ -541,7 +541,14 @@ class GCodeExport : public NoCopy
 
     void writeFanCommand(double speed);
 
-    void writeTemperatureCommand(const size_t extruder, const Temperature& temperature, const bool wait = false);
+    /*!
+     * \brief Write a GCode temperature command
+     * \param extruder The extruder number
+     * \param temperature The temperature to bo set
+     * \param wait Indicates whether we should just set the temperature and keep going, or wait for the temperature to be reach before going further
+     * \param force_same_temperature When true, we should set the temperature command even if the actual set temperature is the same
+     */
+    void writeTemperatureCommand(const size_t extruder, const Temperature& temperature, const bool wait = false, const bool force_same_temperature = false);
```

Comment:
I think to me, `force_same_temperature` implies that some temperature value is being forced, rather than that the _writing_ happens when the temperature _is_ the same. Maybe `force_write_on_equal`?